### DISCUSSION TITLE
Zadeklaruj swoją rolę w grupie tłumaczy

### DISCUSSION THREAD

**Discussion by zwiastunsw** [irdpl/wcag#26] [Organizacja pracy]
Potrzebujemy redaktorów-opiekunów KS i redaktorów - opiekun terminów słownikowych. Aktualnie mamy "17 sierotek": 10 kryteriów i 7 terminów. Można opiekować się więcej niż jednym kryterium i powiązanymi z nim terminami.

Zadeklaruj swoją rolę w tym wątku. Terminy słownikowe powiązane z KS są odnotowane w wątkach wprowadzających.

**Comment by zwiastunsw** [comment]
Na razie mamy zgłoszone:
- @BartekStepien: kryteria dotyczące fokusu (+powiązane terminy słownikowe)
- @PiotrtWitek: Zbędne wpisy (+ powiązane terminy słownikowe) lub Spójna pomoc (+ powiązane terminy słownikowe)
- @zwiastunsw: Spójna pomoc (+ powiązane terminy słownikowe)

**Comment by PiotrtWitek** [comment]
Dzięki za podsumowanie. Coraz bardziej mi się ten Git podoba. ;)
Wiadomość napisana przez Stefan Wajda ***@***.***> w dniu 07.03.2023, o godz. 14:20:


Na razie mamy zgłoszone:
    • @BartekStepien: kryteria dotyczące fokusu (+powiązane terminy słownikowe)
    • @PiotrtWitek: Zbędne wpisy (+ powiązane terminy słownikowe) lub Spójna pomoc (+ powiązane terminy słownikowe)
    • @zwiastunsw: Spójna pomoc (+ powiązane terminy słownikowe)

**Reply by rotnicki** [reply]
@PiotrtWitek 
Mam taką uwagę i sugestię: jeśli odpowiadasz za pomocą e-maila to warto wyczyścić ją z wszelkich niepotrzebnych cytowań przed odesłaniem.
Bo to wszystko wchodzi do twojego komentarza - tak jak powyżej:

**Comment by rotnicki** [comment]
Jako, że moim "konikiem" jest dobra widoczność i kontrasty to mógłbym zaopiekować się tłumaczeniem kryterium 2.4.11 Wygląd fokusu. Mocno wczytałem się już w objaśnienia (understanding) do tego kryterium, swoją drogą chyba najbardziej obszerne i złożone "understanding" w całym dotychczasowym WCAG - bo liczy kilkanaście stron z konkretnymi przykładami, schematami i obrazkami. A też uważam, że uprzednie przeczytanie objaśnień do każdego kryterium powinno być obowiązkiem każdego chcącego tłumaczyć jakiekolwiek kryteria WCAG.

**Comment by IzabelaMrochen** [comment]
Jako nowy członek zajmę się kryterium sukcesu 2.5.8

**Comment by PiotrtWitek** [comment]
Jasne Mikołaju. Każda podobna sugestia mile widziana. :)

**Comment by JacZad** [comment]
Miałem chrapkę na 3.3.7, ale widzę innych chętnych. To ja przygarnę 3.3.8 i 3.3.9. Może być?

**Reply by rotnicki** [reply]
@JacZad zgodnie z twoim rzyczeniem przypisałem cię właśnie jako opiekuna do #14 i #15

**Comment by mkscv0** [comment]
Dzień dobry,
Zgłaszam się do roli: redaktor - opiekun KS (kryterium sukcesu).
